api_app/views.py

```diff
@@ -15,52 +15,15 @@ class CustomUserPageNumberPagination(PageNumberPagination):
     page_size_query_param = 'page_size'
 
 
-class CustomUserListView(ListAPIView):  # Add Pagination
+class CustomUserViewSet(ModelViewSet):
```

Comment:
może dodaj pagination_class do całego widoku?

Reply:
Czemu do całego widoku? czy nie lepiej byłoby tylko do listy?

Reply:
class GenericAPIView(views.APIView):
....
  # The style to use for queryset pagination.
  pagination_class = api_settings.DEFAULT_PAGINATION_CLASS
...
    def paginator(self):
 ''The paginator instance associated with the view, or `None`.''

ogólnie paginacje definiuje się per projekt a definiowanie na konkretnym widoku zwykle swiadczy o wyjątku